apiary.apib

```diff
@@ -244,41 +305,301 @@ Lists all available commands (and subcommands) that are available.
 
 # Group config
 
+## config [GET /config{?arg1,arg2}{&bool,json}]
+
+ipfs config controls configuration variables. It works
+much like 'git config'. The configuration values are stored in a config
+file inside your IPFS repository.
+
+**Examples:**
+
+Get the value of the 'datastore.path' key:
+
+    `ipfs config datastore.path`
+
+Set the value of the 'datastore.path' key:
+
+    `ipfs config datastore.path ~/.ipfs/datastore`
+
++ Parameters
+    + arg1: "Addresses.API" (string, required) - The key of the config entry
+    + arg2 (string, required) - The value to set the config entry to
+    + bool (boolean, optional) - Set a boolean
+    + json (boolean, optional) - Parse stringified JSON
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + `Key`: "API.HTTPHeaders" (string)
+        + `Value`: null (string, nullable)
+
++ Response 500 (application/json)
+
+    + Attributes (object)
+        + `Message` (string)
+        + `Code`: 0 (number)
+
 ## edit [GET /config/edit]
+Opens the config file for editing in $EDITOR
 
-+ Response 200 (text/plain)
+To use 'ipfs config edit', you must have the $EDITOR environment
+    variable set to your preferred text editor.
 
-## replace [GET /config/replace]
++ Response 200 (buffer)
 
-+ Response 200 (text/plain)
+## replace [GET /config/replace{?arg}]
+Replaces the config with <file>
+
+Make sure to back up the config file first if neccessary, this operation
+can't be undone.
+
++ Parameters
+
+    + arg (string, required) - The file to use as the new config
```

Comment:
It works with a file path. Added a `curl` example, and example headers. This should be a 204, not a 200. See #14.